docs/cli/templates.md

```diff
@@ -270,7 +270,7 @@ The primary Python template uses Alpine Linux as a runtime environment due to it
 If you need to use pip modules that require compilation then you should try the python3-debian template then add your pip modules to the `requirements.txt` file.
 
 ```
-$ faas-cli template pull https://github.com/openfaas-incubator/python3-debian
+$ faas-cli template pull
```

Comment:
This command won't be needed anymore when using the default templates. Can you update to remove it? I'll merge this now if you like, but it should be changed.